peachjam/templates/peachjam/_related_documents.html

```diff
@@ -38,4 +38,10 @@ <h4>{% translate verb as verb %}{{ verb|capfirst }}</h4>
     </div>
   {% endif %}
 </div>
-{% block extra-related-documents %}{% endblock %}
+{% if user.is_authenticated %}
+  {% block extra-related-documents %}{% endblock %}
+{% else %}
+  <p>
```

Comment:
@Sandravaphilips this needs to be fully translatable. You'll need to store the URLs in variables and then use a blocktrans tag.